### PR TITLE
fix: handle null

### PR DESCRIPTION
**Pull request by dejwi**
### Task Description
[SCOUT-237](https://playmakerpro.atlassian.net/browse/SCOUT-237?atlOrigin=eyJpIjoiM2Q4MWE3YmU0N2M0NDFjZjg1YTdjZjg1N2JkNGY1ZjkiLCJwIjoiaiJ9)

- fixed (hopefully) single player page

I've had a similar problem once before but I couldn't replicate it

### CODE REVIEW

File: src/pages/players/[slug].tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const PlayerPage = ({ data, errorMessage, errorStatus }: TSsrRole<TData>) => {
   const { t } = useTranslation()
   const router = useRouter()
 
-  const { isAdmin, player } = data as TData
+  const { isAdmin, player } = (data as TData) || {}
```
Comment (jakubfiglak) [inline src/pages/players/[slug].tsx:40]:
```suggestion
  const { isAdmin, player } = data || {}
```

You don't need to cast this type anymore. In fact, casting it was a mistake in the first place, there's a reason why typescript complains about it - data can be null.

**Review by jakubfiglak — COMMENTED** [review]
I don't understand the issue. Were you able to replicate the problem they showed in the ticket?

File: src/pages/players/[slug].tsx (hunk 1 above)
Comment (dejwi) [reply src/pages/players/[slug].tsx]: okay now I've replicated it😅
`|| {}` helped with this:
image
so we could show this:
image

and the primary problem was fixed in handle slug backend pr that you already saw🙏

Comment (jakubfiglak) [reply src/pages/players/[slug].tsx]: Oh, ok. Got it

**Review by jakubfiglak — APPROVED** [review]

**Pull request merged**